src/app/Controllers/talking.component.ts

```diff
@@ -137,7 +136,7 @@ export class TalkingComponent implements OnDestroy, AfterViewInit {
         }
     }
 
-    public ngAfterViewInit(): void {
+    public ngOnInit(): void {
         const inputElement = <HTMLElement>document.querySelector('#chatInput');
```

Comment:
Move L141-L149 to ngAfterViewInit to prevent null pointer.
in ngOnInit the dom element might not be inited immediately